srcStatic/IniFile.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <string>
+
+
+class IniSection;
+
+
+// Represents an entire .ini file
+class IniFile {
+public:
+
+	IniFile(std::string fileName);
```

Comment:
I would have preferred to see filename for the various argument and variable instance names (treating filename as one word instead of two words, file name).

Reply:
I thought about that. I figured you might mention it. Though after writing "keyName" and "sectionName" so many times, it just felt wrong to write anything other than "fileName". In this context, I am trying to distinguish between different types of names, one of which happens to be of files. As such, I would consider this a non-compound form of "file name".